.circleci/config.yml

```diff
@@ -0,0 +1,23 @@
+version: 2
+jobs:
+  build:
+    docker:
+      - image: circleci/node:8-browsers
```

Comment:
Old stuff

Reply:
Would go to node 10, and I guess leave mongo a little old, since this does work with typical 2.x projects